src/geode/geometry/intersection.cpp

```diff
@@ -494,6 +491,327 @@ namespace geode
         return line_intersection_result.type;
     }
 
+    IntersectionResult< absl::InlinedVector< Point3D, 2 > >
```

Comment:
Maybe cut this into several sub-functions depicting the different cases, with variables with more meaningful names

Reply:
I agree but it would be preferable to conserve the initial code